components/tokens/texts/textinfo.vue

```diff
@@ -1,13 +1,13 @@
 <template>
-  <p><slot /></p>
+  <div><slot /></div>
```

Comment:
why remove the `p` tag for a `div`? doesn't it have more sense with a `p` and also help accessibility?

Reply:
You can use `inline-block` if you want you `p` to behave like a `div`

Reply:
Because it's not really a text paragraph for me. But yeah i have no opinion on that i will change